Sources/DefaultsController.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+enum UserDefaultsState: Equatable {
+    case standard
+    case appGroup(String)
+
+    static func makeState(appGroupID: String?) -> UserDefaultsState {
```

Comment:
`init(appGroupID: String?)` ?

Reply:
I actually don't think you need this at all, consumers should choose to either use `standard` or `appGroup("blah")`